src/main/java/com/redhat/devtools/intellij/quarkus/module/QuarkusExtensionsStep.java

```diff
@@ -124,18 +107,23 @@ public QuarkusExtensionsStep(WizardContext wizardContext) {
 
     @Override
     public JComponent getComponent() {
-        if (panel == null && wizardContext.getUserData(QuarkusConstants.WIZARD_EXTENSIONS_MODEL_KEY) != null) {
-            panel = new JPanel();
+        if (outerPanel == null && wizardContext.getUserData(QuarkusConstants.WIZARD_EXTENSIONS_MODEL_KEY) != null) {
+            outerPanel = new JPanel();
+            outerPanel.setLayout(new BorderLayout());
+
+            JPanel panel = new JPanel();
             panel.setLayout(new BoxLayout(panel, BoxLayout.Y_AXIS));
+            panel.setBorder(JBUI.Borders.empty(20));
+
+
             JLabel label1 = new JLabel("Filter extensions");
             label1.setAlignmentX(Component.LEFT_ALIGNMENT);
             panel.add(label1);
             SearchTextField filter = new SearchTextField() {
                 @Override
                 public Dimension getMaximumSize() {
-                    Dimension size = super.getPreferredSize();
-                    size.height = JBUI.scale(30);
-                    return size;
+                    Dimension maxSize = super.getMaximumSize();
+                    return new Dimension(maxSize.width, JBUI.scale(30));
```

Comment:
Made the filter field fill the horizontal space:

<img width="912" alt="Screenshot 2023-06-26 at 10 21 45" src="https://github.com/redhat-developer/intellij-quarkus/assets/148698/bfc9b3ec-a037-4832-b508-678484296152">